spec/mastermind_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './mastermind.rb'
+describe Mastermind do
```

Comment:
Think about adding the SimpleCov gem, https://github.com/colszowka/simplecov ... actually think about adding a Gemfile for this project (for all projects really 😄 ).

Reply:
SimpleCov will let you know which lines are/aren't covered by your tests.

Reply:
oooh that's awesome!!